src/Faker/Provider/nl_NL/Person.php

```diff
@@ -2,6 +2,84 @@
 
 namespace Faker\Provider\nl_NL;
 
+use Faker\Provider\DateTime;
+
+abstract class IdentityDocumentDate
+{
+    protected $date;
+    protected $validity;
+
+    /**
+     * @param \DateTime|null $date     Issue or Expiration date
+     * @param \DateTime|null $birth    Birthdate of person
+     * @param string|null    $timezone Timezone to use
+     */
+    public function __construct(\DateTime $date = null, \DateTime $birth = null, $timezone = null)
```

Comment:
I don't see a value to use these ValueObjects in a faker provider. While it makes sense to use such objects and DDD approach in application business logic, it is making it more difficult to use Faker and is not consistent with the rest of providers. Faker was designed to offer a quick way to generate some data in an easy way. Moreover, there is no logical dependence between the generated number and expiry date or birth date. The only logic here is validity period depending on the birth date and issue date which could be resolved in client app.

Reply:
Thanks for your suggestions. I will indeed refactor and stick to the return of the documentnumber only.

I can add a method for calculating the validity (since that could be different per locale?) and pass `age` as an integer (age might be known, but not someone's birthdate), but the calculation of issue & expiry dates can be done by the client outside Faker.

Reply:
> Thanks for your suggestions. I will indeed refactor and stick to the return of the documentnumber only.
> 
> I can add a method for calculating the validity (since that could be different per locale?) and pass `age` as an integer (age might be known, but not someone's birthdate), but the calculation of issue & expiry dates can be done by the client outside Faker.

Done

Reply:
`DateTimeInterface` should probably be preferred over `DateTime`. Not sure how many other places we have this. We can safely change any param types and phpdoc without it being a breaking change (outside of this PR).

Reply:
Apparently we can't change it everywhere, but I will adjust it here. Thanks!